### PR TITLE
Add querys custom

### PR DESCRIPTION
**Pull request by carolinaWolox**
## Summary
- Add querys custom
## Trello Card
https://trello.com/c/ObwrBKrL/30-queries-custom-en-repositorio

### CODE REVIEW

File: src/main/java/wolox/training/controllers/BookController.java [hunk 1]
```diff
@@ -70,9 +70,10 @@ public String greeting(@RequestParam(name = "name", required = false, defaultVal
         return bookRepository.findByTitle(bookTitle);
     }
 
-    @GetMapping("/publisherGenreYear/{publisher}/{genre}/{year}")
-    public List<Book> findByGenrePublisherYear(@PathVariable String publisher, @PathVariable String genre,
-            @PathVariable String year) {
+    @GetMapping("/publisherGenreYear")
```
Comment (juanwongramirez) [inline src/main/java/wolox/training/controllers/BookController.java]: You can use the root path "/", try to use a less descriptive paths

File: src/main/java/wolox/training/controllers/UserController.java [hunk 2]
```diff
@@ -97,11 +97,17 @@ public UserBook findByName(@PathVariable String name) {
     }
 
     @GetMapping("/birthdayAndName")
```
Comment (juanwongramirez) [inline src/main/java/wolox/training/controllers/UserController.java]: You can use the root path "/", try to use a less descriptive paths

**Review by juanwongramirez — CHANGES_REQUESTED** [review]

File: src/main/java/wolox/training/controllers/BookController.java (hunk 1 above)
Comment (carolinaWolox) [reply src/main/java/wolox/training/controllers/BookController.java]: fixed at last commit

File: src/main/java/wolox/training/controllers/UserController.java (hunk 2 above)
Comment (carolinaWolox) [reply src/main/java/wolox/training/controllers/UserController.java]: fixed at last commit

File: src/main/java/wolox/training/controllers/UserController.java [hunk 3]
```diff
@@ -90,18 +90,24 @@ public Object getLoggedUser(Authentication authentication) {
      *
      * @return {@link UserBook} book's user
      */
-    @GetMapping("/name/{name}")
+    @GetMapping("/{name}")
```
Comment (julianvasquez-wolox) [inline src/main/java/wolox/training/controllers/UserController.java]: for this endpoint, the /name was ok, since the name param is always required

**Review by juanwongramirez — APPROVED** [review]
God job!!

File: src/main/java/wolox/training/controllers/UserController.java (hunk 3 above)
Comment (carolinaWolox) [reply src/main/java/wolox/training/controllers/UserController.java]: Fixed at last commit !

**Review by julianvasquez-wolox — APPROVED** [review]
Let's go! 🙌🏼

**Pull request merged**